src/modules/explorer/pages/DAOList/index.tsx

```diff
@@ -126,24 +188,22 @@ export const DAOList: React.FC = () => {
                   <Grid item>
                     <StyledTab
                       variant="contained"
-                      color={selectedTab !== 0 ? "primary" : "secondary"}
                       style={selectedTab !==0 ? {borderTopRightRadius: 0, borderBottomRightRadius: 0, zIndex:0} : {borderRadius: 4, zIndex: 1}}
-                      disableRipple={true}
                       disableElevation={true}
                       onClick={() => handleChangeTab(0)}
+                      isSelected={selectedTab === 0 ? true : false}
                     >
                       All
                     </StyledTab>
                   </Grid>
                   <Grid item>
                   
                     <StyledTab
-                      disableRipple={true}
                       disableElevation={true}
                       variant="contained"
-                      color={selectedTab !== 1 ? "primary" : "secondary"}
                       style={selectedTab !==1 ? {borderTopLeftRadius: 0, borderBottomLeftRadius: 0, marginLeft: -1, zIndex: 0} : {borderRadius: 4, marginLeft: -1, zIndex: 1}}
                       onClick={() => handleChangeTab(1)}
+                      isSelected={selectedTab === 1 ? true : false}
```

Comment:
Instead of doing a ternary we could do just: `selected === 1` as it will return a boolean